README.md

```diff
@@ -24,7 +24,13 @@ You'll need [Access Keys](https://docs.aws.amazon.com/powershell/latest/userguid
 
 ## Environment variables
 
-For envirnoment variables in your app, you can provide a `repo_env` file in your repo, a `.env` file in GitHub Secrets named `DOT_ENV`, or an AWS Secret. Then hook it up in your `docker-compose.yaml` file like:
+For envirnoment variables in your app, you can provide:
+ - `repo_env` file in your repo
+ - `GH_vars` file in GitHub Variables
+ - `.env` file in GitHub Secrets
+ - `AWS Secret` in JSON format
```

Comment:
the syntax of the code snippets is inconsistent (e.g. `repo_env` vs `GH_vars` vs `AWS Secret`.  I think consistency would be good here.  how about
```suggestion
 - `repo_env` - A file in your repo that contains env vars
 - `ghv_env` An entry in GitHub Action Variables
 - `dot_env` An entry in GitHub Secrets
 - `aws_secret_env` The path to a JSON format secret in AWS
```